src/overlays/actors/ovl_Bg_Spot16_Bombstone/z_bg_spot16_bombstone.h

```diff
@@ -3,12 +3,27 @@
 
 #include <ultra64.h>
 #include <global.h>
+#include "overlays/actors/ovl_En_Bombf/z_en_bombf.h"
```

Comment:
maybe the .c should include `z_en_bombf.h` instead of the .h, the .h has no use for it? not sure